server_installation/topics/operator/keycloak-user-cr.adoc

```diff
@@ -6,7 +6,7 @@ You can use the Operator to create users in {project_name} as defined by a custo
 
 [NOTE]
 ====
-You can update properties, except for the password, in the YAML file and changes appear in the {project_name} admin console, however changes to the admin console do not update the custom resource.
+You can create and delete the YAML file and changes appear in the {project_name} admin console, however changes to the admin console are not reflected back and updates of the CR are not supported.
```

Comment:
This is not accurate. Client and User support limited updatability.

Reply:
removed